politeiawww/comments.go

```diff
@@ -21,6 +21,31 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+// counters is a struct that helps us keep track of up/down votes.
```

Comment:
Any suggestions?

Reply:
Sure. Delete that code since it is confusing and adds no value. Then do the math in where it is currently calling these functions. This should make the code way more obvious.

Reply:
Once you folks decide that this change/PR is worthwhile and should be merged I can fix the code as suggested.

Is there some other bug / feature for a decred beginner that I could tackle in the meantime?

Reply:
I have simplified the code and moved it closer to where it's used. I hope it is plain enough now :)

Reply:
Get rid of the add/subtract functions and just do the math in place.  Calling functions for that just makes it harder to understand what is actually going on.

Reply:
Done.